apis/python/src/tiledbsoma/io/ingest.py

```diff
@@ -63,6 +64,7 @@ def from_h5ad(
     platform_config: Optional[PlatformConfig] = None,
     ingest_mode: IngestMode = "write",
     use_relative_uri: Optional[bool] = None,
+    sparsify_X: bool = True,
```

Comment:
Can I suggest taking a page from @thetorpedodog and making this more flexible? Ideally, the user could specify what type they want to use for X  layers.

Proposed argument: make it match _either_ `tiledbsoma.open` which uses `soma_type`, or `tiledbsoma.Collection.add_new_collect` which uses `kind`.

I.e.,     `X_kind: Union[Type["_tiledb_object.AnyTileDBObject"], str, None] = None`

Reply:
done